### ISSUE TITLE
Attach any document (image, audio) causes crash with message: Unfortunately, Documents has stopped.

### ISSUE THREAD

**Issue by jgardner04** [WhisperSystems/Signal-Android#5465]
I have:
- [x] searched open and closed issues for duplicates
- [x] read https://github.com/WhisperSystems/Signal-Android/wiki/Submitting-useful-bug-reports

----------------------------------------

### Bug description
Trying to add Image, Audio, or Video on my Nexus 6p causes the document to crash with an error message of "Unfortunately, Documents has stopped"

### Steps to reproduce
- Open Signal
- Open a message thread
- Select the attachment icon
- Select an image, audio, or video

**Actual result:** Signal opens document selection but crashes before anything can be seen.  Signal resets and I am able to type another message.
**Expected result:** I should be able to select and image, audio file, or video and insert it into the chat window.

### Screenshots
I am not able to capture the popup message with a screen shot.  It only captures what is under it. 

### Device info
**Device:** Hauwei Nexus 6P (angler)
**Android version:** 6.0.1 (2705526, MHC19Q)
**Signal version:** 3.15.2

### Link to debug log
[Debug Log](https://gist.github.com/anonymous/8d52fdeceb697763ff72d6409cef301d)

### Additional Information
I am able to share a file by opening Google Photos or other application and share it with Signal.  I am still able to share contacts, camera, and location as expected. 

I was able to briefly get around this issue by going to Settings->Storage & USB->Explore and opening the pictures folder.  Doing this allowed me to select a picture in Signal but only one time.  The crash behavior returns and this has not been repeatable.

**Comment by 2-4601** [comment]
There doesn't seem to be any crashes or exceptions in the log you posted. Are you sure it's a Signal issue? What is _**Documents**_? Can you check that you have granted Signal all the permissions it requires (Android M permission control etc.)?

**Comment by jgardner04** [comment]
@2-4601 It looks like I have some more information.  After further research I ran across and article regarding a different [application that was exhibiting a similar issue](https://productforums.google.com/forum/m/#!topic/nexus/BtlKhzk50Ik;context-place=topicsearchin/nexus/category$3Anexus-6p). I also have OneDrive installed and I work for MSFT so I have it attached to a corporate signin.  When I remove my corporate account, I am attach images, audio, and video as is the expected behavior.  I can confirm that adding the corporate account back to the OneDrive application repeats the unexpected behavior.   

I did capture the Signal log but my guess is that this is an issue with Android not Signal based on the testing above..
